src/components/Book.js

```diff
@@ -15,28 +18,44 @@ const Book = () => {
 
   const booksArray = Object.values(books)[0];
   return (
-    <tbody className="table_body">
+    <>
       {booksArray.map((book) => (
-        <tr key={book.id}>
-          <td>
-            {book.title}
-            {' '}
-            by
-            {' '}
-            {book.author}
-          </td>
-          <td>
-            <button
-              type="button"
-              className="remove_btn"
-              onClick={() => handleRemoveBook(book.id)}
-            >
-              Remove
-            </button>
-          </td>
-        </tr>
+        <section className="each-book" key={book.id}>
+          <div className="bk-details">
+            <h5 className="entry-category">Category</h5>
+            <h3 className="entry-title">{book.title}</h3>
+            <h5 className="entry-author">{book.author}</h5>
+            <div className="bk-actions">
+              <button type="button" className="btn-type1">Comments</button>
+              <hr className="vertical line-1" />
+              <button
+                type="button"
+                className="remove_btn btn-type1"
+                onClick={() => handleRemoveBook(book.id)}
+              >
+                Remove
+              </button>
+              <hr className="vertical line-1" />
+              <button type="button" className="btn-type1">Edit</button>
+            </div>
+          </div>
+          <div className="progress-cont">
+            <CircularProgressbar />
+          </div>
+          <div className="stats">
```

Comment:
Good job taking care of the spaces at the right 👍. However, while addressing that issue I  realized that you have added more issues, the progress bar now collides with the text and the buttons are now looking squished up. I perceive the resolution of your PC is quite big and the design is looking good on your side, so I am recommending you test it out with a resolution of width `1300` and below which is my range.
I recommend that this issue be addressed in order to provide a consistent UI for users on different desktop screen sizes.


Here is what i see

![bookstore](https://user-images.githubusercontent.com/84930355/222918267-459f3d49-1edf-4b03-80e3-2da60cd739f8.PNG)

Reply:
Thanks, I've made changes to make it work with a minimum desktop width of 1200px, so it should be fine now.